Modules/_ctypes/cfield.c

```diff
@@ -71,6 +71,16 @@ PyCField_FromDesc(PyObject *desc, Py_ssize_t index,
         Py_DECREF(self);
         return NULL;
     }
+
+#ifndef MS_WIN32
+    if (pack && bitsize) { /* packed bitfield */
+        size = (bitsize - 1) / 8 + 1;
```

Comment:
(copying the old comment, so that it shows up in the github editor for reviewers)

If we have packed bitfield, calculate the minimum number of bytes needed to fit the bitfield.

Reply:
I'd follow this rule: if a comment is necessary/helpful/crucial for reviewers it should be in the source code so that everyone looking at this code at any point later on sees it, not in GitHub comments.